t/data/tests/tests/shutdown.pm

```diff
@@ -5,8 +5,13 @@ use Mojo::Base 'basetest', -signatures;
 use testapi;
 
 sub run ($) {
-    enter_cmd 'sudo poweroff';
+    power 'off';
     assert_shutdown(get_var('INTEGRATION_TESTS') ? 90 : undef);
+    sleep 8; # TODO this causes autotest to take a little bit longer causing: [2025-02-17T13:05:51.891245+01:00] [warn] [pid:25301] !!! OpenQA::Isotovideo::Runner::_read_response: THERE IS NOTHING TO READ 17 4 3
```

Comment:
Given the much quicker execution without this I see this as something temporary you can try out to experiment with but not anything acceptable for merging